src/AbstractMenu.js

```diff
@@ -45,8 +45,12 @@ export default class AbstractMenu extends Component {
             case 13: // enter
                 e.preventDefault();
                 this.tryToOpenSubMenu(e);
-                if (this.seletedItemRef && this.seletedItemRef.ref instanceof HTMLElement) {
+                if (this.seletedItemRef &&
+                    this.seletedItemRef.ref instanceof HTMLElement &&
+                    !this.seletedItemRef.ref.props.disabled) {
```

Comment:
AFAIK if `this.seletedItemRef.ref instanceof HTMLElement` is `true` , `this.seletedItemRef.ref.props` will not exist since it will not refer to a `ReactElement`. Correct me if I'm wrong,

Reply:
not exactly, I think. Below is the console output of this.selectedItemRef:
<img width="667" alt="screen shot 2017-11-29 at 1 56 01 pm" src="https://user-images.githubusercontent.com/34054635/33360009-af419dce-d50c-11e7-9e9b-0c73c7295e6b.png">

Reply:
Can you please, also show the output for `this.seletedItemRef.ref instanceof HTMLElement`??

Reply:
```
      case 13:
        // enter
        e.preventDefault();
        _this2.tryToOpenSubMenu(e);
        console.log("handleKeyNavigation es6");
        console.log(_this2.seletedItemRef);
        if (_this2.seletedItemRef) {
          console.log("_this2.seletedItemRef.ref instanceof HTMLElement");
          console.log(_this2.seletedItemRef.ref instanceof HTMLElement);
        }

        if (
          _this2.seletedItemRef &&
          _this2.seletedItemRef.ref instanceof HTMLElement &&
          !_this2.seletedItemRef.props.disabled
        ) {
          _this2.seletedItemRef.ref.click();
        } else {
          console.log("hideMenu");
          _this2.hideMenu(e);
        }
        break;
```
<img width="657" alt="screen shot 2017-11-29 at 2 08 36 pm" src="https://user-images.githubusercontent.com/34054635/33360377-6a3d4230-d50e-11e7-8eb7-0dadff4b85df.png">

Reply:
`this.seletedItemRef.ref instanceof HTMLElement` is true

Reply:
@vkbansal I am afraid I made a mistake in this PR. The code change in my local to determine the disabled status of the menuitem is ` !_this2.seletedItemRef.props.disabled`. But there was a typo in the PR: `!this.seletedItemRef.ref.props.disabled`. I will create another PR.

Reply:
@WaterDesk  no issues